### PR TITLE
[CI] Add setup install fs required services script

### PR DESCRIPTION
**Pull request by eliyahu77**
add dev_utilities script that  uninstall and install (redis, kafka, and mysql)  using  helm

### CODE REVIEW

**Review by yuribros1974 — APPROVED** [review]

File: automation/system_test/prepare.py [hunk 1]
```diff
@@ -331,6 +332,22 @@ def _override_mlrun_api_env(self):
...
+        # make provctl executable
+        self._run_command("rm", args=["-rf", "/home/iguazio/dev_utilities"])
+        self._run_command("git", args=["clone", urlscript, "dev_utilities"])
+        self._run_command("python3", args=["dev_utilities.py", "uninstall", "--redis", "--mysql", "--redisinsight", "--kafka"], workdir="/home/iguazio/dev_utilities")
+        self._run_command("python3", args=["dev_utilities.py", "install", "--redis", "--mysql", "--redisinsight", "--kafka", ipaddr], workdir="/home/iguazio/dev_utilities")
+
+
+
+
+
```
Comment (liranbg) [inline automation/system_test/prepare.py:350]: no need many empty lines, keep file formatted

File: automation/system_test/prepare.py [hunk 2]
```diff
@@ -331,6 +332,22 @@ def _override_mlrun_api_env(self):
             args=["apply", "-f", manifest_file_name],
         )
 
+
+    def _install_devutilities(self):
+
+        urlscript= ("https://gist.github.com/a51d75fe52e95df617b5dbb983c8e6e1.git")
```
Comment (liranbg) [inline automation/system_test/prepare.py:338]: where is the content of this file being versioned?

File: automation/system_test/prepare.py [hunk 3]
```diff
@@ -331,6 +332,22 @@ def _override_mlrun_api_env(self):
...
+        ipaddr = ("--ipaddr " + os.environ.get('IP_ADDR_PREFIX'))
+
+        # make provctl executable
```
Comment (liranbg) [inline automation/system_test/prepare.py:341]: remove comment

**Review by liranbg — CHANGES_REQUESTED** [review]
minor

File: automation/system_test/prepare.py (hunk 2 above)
Comment (eliyahu77) [reply automation/system_test/prepare.py:338]: under  gist github

**Review by uriblum46 — APPROVED** [review]

File: automation/system_test/prepare.py [hunk 4]
```diff
@@ -331,19 +333,46 @@ def _override_mlrun_api_env(self):
             args=["apply", "-f", manifest_file_name],
         )
 
+    def _install_devutilities(self):
+        urlscript = "https://gist.github.com/a51d75fe52e95df617b5dbb983c8e6e1.git"
```
Comment (Tankilevitch) [inline automation/system_test/prepare.py:337]: This points to a personal repository snippet, please adjust to point to an organizational gist.

**Review by Tankilevitch — CHANGES_REQUESTED** [review]

**Review by Tankilevitch — APPROVED** [review]

**Pull request merged**